src/lib/__tests__/flashMessage.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  FlashMessage,
+  forceRemoveAll,
+  messageTTL,
+  messageType,
+} from 'lib/flashMessage';
+
+describe('FlashMessage', () => {
+  it(`won't display messages that are already in the queue by default`, () => {
+    for (let i = 0; i < 6; i++) {
+      new FlashMessage(
+        `Yo! Something went wrong.`,
+        messageType.ERROR,
+        messageTTL.MEDIUM
+      );
+    }
+
+    let notificationWrapper = document.querySelector(
+      '.noty_layout'
+    ) as HTMLElement;
+    expect(notificationWrapper.children).toHaveLength(1);
+
+    forceRemoveAll();
+
+    new FlashMessage(
+      `Yo! Something went wrong.`,
+      messageType.ERROR,
+      messageTTL.MEDIUM
+    );
+    new FlashMessage(
+      `Yo! Something went wrong.`,
+      messageType.ERROR,
+      messageTTL.MEDIUM
+    );
+    new FlashMessage(
+      `Yo! Something went wrong.`,
```

Comment:
I see the messageType is different, and so that's why we wanna see 2 notifications instead of 1 in this test. But maybe also change the text here so it's a bit clearer what the intent of this test is?

Or did you want to exercise checking that messageType is part of what is checked when checking if a notification is already in the queue?

Reply:
I added a comment explaining what it's trying to achieve.

Exactly what you said, I want to ensure that the message type is taken into consideration when checking if a message is already in the queue.